src/PKSim.Presentation/Presenters/Compounds/MolWeightGroupPresenter.cs

```diff
@@ -74,8 +76,8 @@ public class MolWeightGroupPresenter : CompoundParameterGroupPresenter<IMolWeigh
 
       private void valueOriginUpdated(ValueOrigin valueOrigin)
       {
-         //This should update the value origin for all related parameters
-         AddCommand(_parameterTask.SetParametersValueOrigin(_molWeightParameters, valueOrigin));
```

Comment:
used to update everything. Now only editable parameters (MolWeight AND halogens)